### PR TITLE
[ADD] Connector Equipment Service

### PR DESCRIPTION
**Pull request by osi-scampbell**
This PR adds connector_equipment_service to the vertical-isp repo

Design: #26 

Depends on https://github.com/OCA/vertical-isp/pull/27

### CODE REVIEW

File: connector_equipment_service/views/agreement_serviceprofile.xml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+<odoo>
+    <record id="agreement_serviceprofile_add_chatter" model="ir.ui.view">
+        <field name="name">agreement.serviceprofile.add.chatter</field>
+        <field name="model">agreement.serviceprofile</field>
+        <field name="inherit_id" ref="agreement_serviceprofile.agreement_serviceprofile_form"/>
+        <field name="arch" type="xml">
+            <xpath expr="//sheet" position="after">
+                <div class="oe_chatter">
+                    <field name="message_follower_ids" widget="mail_followers"/>
+                    <field name="message_ids" widget="mail_thread"/>
```
Comment (max3903) [inline connector_equipment_service/views/agreement_serviceprofile.xml]: @osi-scampbell Let's move this into the agreement_serviceprofile module.

File: connector_equipment_service/models/agreement_serviceprofile.py [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+
+from odoo import _, api, models
+
+
+class AgreementServiceProfile(models.Model):
+    _name = 'agreement.serviceprofile'
+    _inherit = ['agreement.serviceprofile', 'mail.thread']
+    _description = 'Agreement Service Profiles'
```
Comment (max3903) [inline connector_equipment_service/models/agreement_serviceprofile.py]: @osi-scampbell No need for the _name and the _description.

**Comment by dreispt** [comment]
@osi-scampbell Can you rebase on current 12.0, please? That shoudl make Travis green now.

**Comment by osi-scampbell** [comment]
@dreispt Sould be good to go

**Pull request closed without merging**